src/daria/corrections/color/colorcorrection.py

```diff
@@ -169,6 +199,17 @@ def __call__(
         # Ensure a data format which can be used by cv2 etc.
         return corrected_image.astype(np.float32)
 
+    def write_config_to_file(self, path: Path) -> None:
+        """
+        Writes the config dictionary to a json-file.
```

Comment:
Maybe one should add a warning when calling the function and `roi` is a tuple. It won't be stored.